docs/_articles/en/lwc/writing.md

```diff
@@ -29,76 +29,82 @@ To learn more about `@salesforce` modules, see [`@salesforce` Modules](https://d
 
 In this example, VS Code displays the possible static resource names to choose from when completing the `import` statement.
 
-![Static Resource Completion](./images/vscode_lwc_staticresource_trailhead.png)
+![Static Resource Completion](../../../images/vscode_lwc_staticresource_trailhead.png)
```

Comment:
None of these changes are correct. It should be `./images/..`

Reply:
It seems like the images are not showing up in here?
https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/docs/_articles/en/lwc/writing.md
Or is this pushed to somewhere else?

Reply:
Every commit/PR builds a preview site on netlify, you can access it by looking at the PR checks by clicking on the Details link for `netlify/salesforcedx-vscode/deploy-preview`